extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -865,6 +865,7 @@ export class AmpA4A extends AMP.BaseElement {
   viewportCallback(inViewport) {
     if (this.friendlyIframeEmbed_) {
       setFriendlyIframeEmbedVisible(this.friendlyIframeEmbed_, inViewport);
+      this.friendlyIframeEmbed_.setCssAnimPlayingState(inViewport);
```

Comment:
AFAIK yes, maybe the usage will go beyond Ads but everything embedded should be a valid AMP page. @dvoytenko to confirm.